src/Fixtures/index.js

```diff
@@ -1,27 +1,66 @@
 import animationMenageATrois from './AnimationMenageATrois';
 import animationSquare from './AnimationSquare';
 
-export default {
-  /*   const types: {
-    FITNESS: 'fitness',
-    TECHNICAL: 'technical',
-  }, */
+export const types = {
+  FITNESS: 'fitness',
+  TECHNICAL: 'technical',
+};
+
+export const intensity = {
```

Comment:
I think enum names should be plural, like you did for types

Reply:
You are right. Thanks.
I want to upload only the file index.js, is there a easy way to only add this one?